apps/extension/src/ui/hooks/ledger/common.ts

```diff
@@ -49,11 +49,12 @@ export const ledgerNetworks = [
   //   name: "statemint",
   //   genesisHash: "0x68d56f15f85d3136970ec16946040bc1752654e906147f7e43e9d539d7c3de2f",
   // },
-  {
-    name: "centrifuge",
-    genesisHash: "0xb3db41421702df9a7fcac62b53ffeac85f7853cc4e689e0b93aeb3db18c09d82",
-    label: "Centrifuge",
-  },
+  // Removed because no longer maintained https://support.ledger.com/hc/en-us/articles/11158925739677-Centrifuge-CFG-
+  // {
+  //   name: "centrifuge",
```

Comment:
bit worried about what happens for users that already created an account for centrifuge.
might want to add a "legacy" property on entries of this array to prevent account creation, but keep existing accounts behave as before ?

Reply:
Discussed in Discord - result of doing this is that existing accounts can still be viewed and balances are fetched, but transactions are not able to be signed with them, and new accounts are not able to be created for the network.